Assignment 5.Rmd

```diff
@@ -57,6 +57,20 @@ plot(score_ctree)
 ```
 
 Please interpret the tree, which two behaviors do you think the teacher should most closely pay attemtion to?
+ They should look at hints prior percent correct
```

Comment:
It would be good to know why you think this is true? How did you reach this conclusion?